### PR TITLE
ZNC 1.9.1

### PR DESCRIPTION
**Pull request by DarthGandalf**
Fixes CVE-2024-39844

### CODE REVIEW

**Comment by tianon** [comment]
So fresh the CVE isn't even public yet! :smile:
(https://wiki.znc.in/ChangeLog/1.9.1 has good links for the curious / those looking for mitigation tips)

File: library/znc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Maintainers: Alexey Sokolov <alexey+znc@asokolov.org> (@DarthGandalf)
 GitRepo: https://github.com/znc/znc-docker.git
-GitCommit: 8638ba43acaef4bff2d4d39668c1992f411230a6
+GitCommit: 14890ffcd1006eefbe0e2aeb9ea389b49eb995ce
 Architectures: amd64, arm32v6, arm64v8
 
 Tags: 1.9.0, 1.9, latest
```
Comment (tianon) [inline library/znc:6]:
```suggestion
Tags: 1.9.1, 1.9, latest
```

:eyes:

**Comment by DarthGandalf** [comment]
Thanks for quick review!

File: library/znc [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 GitRepo: https://github.com/znc/znc-docker.git
-GitCommit: 8638ba43acaef4bff2d4d39668c1992f411230a6
+GitCommit: 14890ffcd1006eefbe0e2aeb9ea389b49eb995ce
 Architectures: amd64, arm32v6, arm64v8
 
-Tags: 1.9.0, 1.9, latest
+Tags: 1.9.1, 1.9, latest
 Directory: full
 
 Tags: 1.9.0-slim, 1.9-slim, slim
```
Comment (tianon) [inline library/znc:9]: Gah, should've expanded the context:

```suggestion
Tags: 1.9.1-slim, 1.9-slim, slim
```

**Review by tianon — CHANGES_REQUESTED** [review]

**Review by tianon — APPROVED** [review]
This time, this is the one! :smile: :heart:

**Pull request merged**